feature/eks-cluster-creation/vars.tf

```diff
@@ -0,0 +1,60 @@
+// -- [ VARS ] -------------------------------------------------------------------------
```

Comment:
can you add comment for each variable in this file to describe it with one sentence what is it for?

Reply:
Done.